kafka-3.8.yaml

```diff
@@ -50,13 +58,16 @@ pipeline:
       rm -rf ${{targets.destdir}}/usr/lib/kafka/windows
 
 subpackages:
-  - name: kafka-bitnami-compat
+  - name: kafka-bitnami-compat-${{vars.major-minor-version}}
     description: "compat package with bitnami/kafka image"
+    dependencies:
+      provides:
+        - kafka-bitnami-compat=${{package.full-version}}
     pipeline:
       - uses: bitnami/compat
         with:
           image: kafka
-          version-path: 3.6/debian-12
+          version-path: 3.8/debian-12
```

Comment:
Can we use ${{vars.major-minor-version}} here?

Reply:
We can't use it here unfortunately. We have to be explicit